app/src/main/java/comp3350/mealbuddy/objects/FoodIntPair.java

```diff
@@ -0,0 +1,23 @@
+package comp3350.mealbuddy.objects;
+
+import java.util.Objects;
+
+public class FoodIntPair {
+
+    public Food food;
+    public Integer quantity;
+
+    public FoodIntPair(Food food, Integer quantity) {
+        this.food = food;
+        this.quantity = quantity;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof FoodIntPair)) return false;
+        FoodIntPair that = (FoodIntPair) o;
+        return food.equals(that.food);
+    }
```

Comment:
do we care if the quantities are equal? cause this doesn't check that

Reply:
No we just want to check the food item.  That is why we implimented our own pair.  Otherwise we could have a banana and a banana with quantity of 2 be in the same list and defeats the purpose of having the quantity pair in the first place